.github/ISSUE_TEMPLATE/eia_new_year.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Integrate New Year of EIA data
+about: Check-list for integrating a new year of EIA data
+title: ''
+labels: new-data
+assignees: ''
+
+---
+
+### EIA new year of data integration check-list:
+
+Review Annual Updates Docs: https://catalystcoop-pudl.readthedocs.io/en/dev/dev/annual_updates.html
+
+Replace references to `EIA_DATASET_NAME` with either `eia_860`, `eia923`, or `eia861` 
+depending on which EIA dataset you are updating.
+
+- [ ] Add the new Zenodo archive DOI to `pudl/workspace/datastore.py`.
+- [ ] Run the datastore script to download the new raw data: `pudl_datastore --dataset 
+EIA_DATASET_NAME`. The new raw data will appear in `pudl_input/eia923/<ZENODO_DOI>/...`
```

Comment:
i think this link is more like: `PUDL_INPUT/data/EIA_DATASET_NAME/ZENODO_DOI/...`

Reply:
thx!!

Reply:
this guy looks the same still/